docs/docs/use_cases/tagging.ipynb

```diff
@@ -318,17 +318,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 5,
    "id": "bf1f367e",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from pydantic import BaseModel, Field"
+    "from pydantic.v1 import BaseModel, Field"
```

Comment:
this assumes they have pydantic v2 installed. maybe we can do `from langchain.pydantic_v1`? or at the very least we should add a comment